cargo-dist/src/backend/ci/github.rs

```diff
@@ -217,10 +243,68 @@ impl GithubCiInfo {
         Ok(rendered)
     }
 
+    /// Generate and write the vendored actions to disk
+    pub fn write_vendored_actions(&self, dist: &DistGraph) -> DistResult<()> {
+        if dist.tools.git.is_none() {
+            warn!("Unable to vendor external actions; git is missing");
+            return Ok(());
+        }
+
+        let path = dist.workspace_dir.join(GITHUB_ACTIONS_DIR);
+
+        vendor_repo(
+            &path,
+            &GithubRepoPair::from_str(GITHUB_VENDORED_DOWNLOAD_ARTIFACT_NWO)
+                .expect("failed to parse string?!"),
```

Comment:
This is based on a static string up at the top of the file, so if it fails it's a bug in cargo-dist rather than something to treat as a runtime error. We do similar `expect`s on cases using `const` values that we don't expect to ever be able to fail elsewhere in cargo-dist.